### PR TITLE
chore: simplify the code

### PR DESCRIPTION
**Pull request by chenquan**

### CODE REVIEW

File: arroyo-api/src/optimizations.rs [hunk 1]
```diff
@@ -44,39 +44,43 @@ fn remove_in_place(graph: &mut DiGraph<StreamNode, StreamEdge>, node: NodeIndex)
...
-                        }
+                let mut ins = graph.edges_directed(idx, Incoming);
+                if !(no_shuffles && in_degree == 1) {
+                    continue;
+                }
+
+                let source_idx = ins.next().unwrap().source();
+                let in_node = graph.node_weight_mut(source_idx).unwrap();
+                if let Operator::Window { agg, .. } = &mut in_node.operator {
+                    if !agg.is_none() {
```
Comment (jacksonrnewhouse) [inline arroyo-api/src/optimizations.rs:67]: It might be cleaner as 
```suggestion
                    if agg.is_some() {
```

**Review by jacksonrnewhouse — COMMENTED** [review]
Thanks for opening this! Can you run `cargo fmt` on it? We have a check that requires the project to be formatted with it.

File: arroyo-api/src/optimizations.rs (hunk 1 above)
Comment (chenquan) [reply arroyo-api/src/optimizations.rs:67]: That's a good idea.

**Comment by chenquan** [comment]
> Thanks for opening this! Can you run `cargo fmt` on it? We have a check that requires the project to be formatted with it.

done.

**Review by jacksonrnewhouse — APPROVED** [review]

**Pull request merged**